webui/src/types.ts

```diff
@@ -6,7 +6,7 @@ export interface TBase {
   examples?: unknown[]
 }
 
-export type TString = TBase & { type: 'string'; is_stream?: boolean; enum?: string[] }
```

Comment:
Should be `;`?

Reply:
This was automatically formatted by eslint.